plenum/server/node.py

```diff
@@ -98,6 +98,7 @@ class Node(HasActionQueue, Motor, Propagator, MessageProcessor, HasFileStorage,
     _client_request_class = SafeRequest
     ledger_ids = [POOL_LEDGER_ID, DOMAIN_LEDGER_ID]
     _wallet_class = Wallet
+    STARTUP_PRIMARY_CONNECTION_CHECK_TIMEOUT = 10
```

Comment:
Probably it makes sense to move the timeout to config (we should eventually re-factor the current config, but let's have all timeouts in one place). Another option is to use the same timeout as for view change (probably reduce it from 60 sec?)